sepal_ui/mapping/draw_control.py

```diff
@@ -51,3 +52,47 @@ def hide(self):
         self not in self.m.controls or self.m.remove_control(self)
 
         return
+
+    def to_json(self):
+        """
+        Return the content of the DrawCOntrol data without the styling properties and using a polygonized representation of circles.
+        The output is fully compatible with __geo_interface__.
+
+        Return:
+            (dict): the json representation of all the geometries draw on the map
+        """
+
+        features = [self.polygonize(feat) for feat in self.data]
```

Comment:
with polygonize, we are mutating each json geometry from the original `DrawingControl`, and as in the next line we are removing the `style` property, and an error will be thrown if you call twice `to_json` method... I tried .`.copy` over the feat dict, but it didn't work, so I used `deepcopy`...

Apparently there is a bug in the `DrawingControl`, if you create more than circles, the firsts will lose their radius, so the `to_json` can't succeed...

Reply:
So as a starter, the feat should not be modified effectively nice catch and I hope the deepcopy do the trick. 
Which version of ipyleaflet are you using ? maybe we need to update to the latest one. Could you try that and let me know if the issue persist ?

Reply:
Using ipyleaflet==0.16.0